### DISCUSSION TITLE
Multiple Exposure is gone

### DISCUSSION THREAD

**Discussion by DA21bad** [sn4k3/UVtools#860] [Q&A]
Hi, just new to this amazing tool, I am trying to calibrate my exposure time but on the exposure time calibration menu there is only the multiple brightness tool, the multiple exposure time is gone.

How can I use these to measure my exposure time correctly

**Comment by sn4k3** [comment]
Only some printers/formats can use it. If not there mean is not compatible with your setup.

**Reply by DA21bad** [reply]
but there is the software, the software do not gives me the option of multiple exposures. I have attached pictures from the menu
![UVTools01](https://github.com/sn4k3/UVtools/assets/110840929/b60818cf-d6e1-4bee-837b-5d7b21523f9f)
![UVTools02](https://github.com/sn4k3/UVtools/assets/110840929/3d82e4fc-a22d-4823-b8f0-183d5275e8b1)

**Reply by sn4k3** [reply]
Multiple brightness is to test gray shades or use it as multiple exposure time if your printer uses Fake/emulated AA (eg: Anycubic printers)
But again, if UVtools does not show you an option or is greyed out, is because is not compatible and can not be used with your setup.
That tool is to generate a test, multiple exposures are a sub-feature of the tool just like multi brightness or pattern. You can still run it as a single test.

**Reply by DA21bad** [reply]
How UV Tools knows my setup? I have not even selected any printer or any profile it is just not there

**Reply by sn4k3** [reply]
UVtools fetch all data from the file you provide to it (File -> Open). Inside that sliced file there are all the information required.
The file format for your printer also dictate which features can be used.

If you are working without opening a file (using the demo file) that's the wrong way to use it. You always need to open a sliced file for your printer.

https://www.youtube.com/watch?v=lRbZw6GbGCI

**Reply by DA21bad** [reply]
I have not uploaded any file
